starlette_context/plugins/base.py

```diff
@@ -52,18 +52,20 @@ def __init__(
         force_new_uuid: bool = False,
         version: int = 4,
         validate: bool = True,
+        error_response: Optional[Response] = None,
     ):
         if version not in self.uuid_functions_mapper:
             raise TypeError(f"UUID version {version} is not supported.")
         self.force_new_uuid = force_new_uuid
         self.version = version
         self.validate = validate
+        self.error_response = error_response
 
     def validate_uuid(self, uuid_to_validate: str) -> None:
         try:
             uuid.UUID(uuid_to_validate, version=self.version)
         except Exception as e:
-            raise ValueError("Wrong uuid") from e
+            raise ValueError(self.error_response or "Wrong uuid") from e
```

Comment:
I think it should just raise `WrongUUIDError(StarletteContextError)` from `errors.py`.

Reply:
Agreed, at least it would be applying a best practice, at best makes it easier to separate issues that might come from user-defined plugin subclasses